README.md

```
@@ -47,15 +47,61 @@
 
 - 입력한 상마 순서에 따라 기물 위치를 초기화한다.
 
-### 이동 커맨드 입력
+### 메인 화면에서 원하는 기능 선택
 
-- move src dst 양식으로 입력해야 한다.
+- `move`, `status`, `end`를 입력 받을 수 있다.
+- 입력 받은 커맨드에 맞는 기능을 수행한다.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)